Cargo.toml

```diff
@@ -32,6 +32,12 @@ rocksdb = { version = "0.14", default-features = false, features = ["snappy"], o
 cc = { version = ">=1.0.64", optional = true }
 socks = { version = "0.3", optional = true }
 hwi = { version = "0.5", optional = true, features = ["use-miniscript"] }
+# required for esplora feature, don't use base64 versions 0.21.1, it requires MSRV 1.60.0
+base64 = { version = "0.21.2", optional = true }
```

Comment:
This line is not needed once we release rust-esplora-client again, which uses base64 0.21.2

Reply:
Thanks for the reminder! I didn't have to wait for a new `rust-esplora-client` since `base64` fixed their MSRV in a patch release.